Season

nagerz opened the pull request:
A couple concerns here. First LeagueStats name needs to change because it was renamed team_stat_collector. But I think there will be an issue because the team stats that this will modify will not be the same teams as are created elsewhere. Now a new instance of each team is going to be made. We're gonna want them to be the same teams so that their stats can be changed and accessed.

PeregrineReed commented:
Last night I fixed the teams all being the same objects throughout the stat tracker, and changed the league stats name, I didn't push from coming into some other issues I'm working through in stat tracker. I'll do a push later with the updates.

nagerz commented:
> Last night I fixed the teams all being the same objects throughout the stat tracker, and changed the league stats name, I didn't push from coming into some other issues I'm working through in stat tracker. I'll do a push later with the updates.

Ok, sounds good, will revisit then

PeregrineReed commented on test/game_test.rb:
```diff
@@ -91,7 +91,7 @@ def test_it_has_a_total_score
 
   def test_it_has_a_score_difference
-    assert_equal -1, @game_1.score_difference
+    assert_equal (-1), @game_1.score_difference
```
Was this not working for you without the parens?

nagerz replied:
Was working. Just got an "ambiguous syntax" message from my linter and thought it might add clarity.

PeregrineReed commented on test/team_stats_test.rb:
```diff
@@ -88,7 +88,8 @@ def setup
     @team_23 = Team.new(team_23_info)
     @team_52 = Team.new(team_52_info)
 
-    @teams = [
```
I think it's fine to keep the bracket on the same line as the @teams =

nagerz replied:
Sure, doesn't matter

PeregrineReed commented on lib/team.rb:
```diff
@@ -44,6 +45,9 @@ def initialize(data)
     }
 
+    #Same thing as above?
+    #@team_info = {data}
```
I switched this when working on the file converter. At first it was returning a csv file line as the data. Also the underscores between the franchise_id, short_name and team_name would have to get removed for the test to pass. If we want to switch it back to how it was that's alright with me. The main issue of the csv file line as the data has been solved.

nagerz commented:
> I also probably should mention that I skipped the tests for the season stats when I was resolving the merge conflicts because the methods weren't written. On that subject, the issues I'm running into with stat tracker are due to the way that the games are being sorted. Preseason games aren't being counted in total games (which in actuality is regular games). There's a few ways that could be fixed, but when the season summary method is written we may have a new way of accessing total games played.
